### PR TITLE
fossil-init: add page

### PR DESCRIPTION
**Pull request by dz-at-tc**
Per issue #6458

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/fossil-init.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# fossil init
+
+> Initialize a new repository for a project.
+> See also: `fossil clone`.
+> More information: <https://fossil-scm.org/home/help/init>.
+
+- Create a new repository in a named file:
+
+`fossil init {{path/to/filename}}`
```
Comment (mfrw) [inline pages/common/fossil-init.md:9]: Should we say repository or directory instead ?
```suggestion
`fossil init {{path/to/repository}}`
```

Comment (kbdharun) [reply pages/common/fossil-init.md:9]: Or would this be better

```suggestion
`fossil init {{path/to/file}}`
```

Comment (dz-at-tc) [reply pages/common/fossil-init.md:9]: Hmmm, as to filename vs file, I went with filename given the style guide [here](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths), but I'm not wed to it.

As to filename vs repository, just looking at the command alone I see your point...  but I wonder if that makes it clear that repositories are in fact single files in fossil (sqlite files under the hood, vs directories like git and svn).

I think the combination of "Create a new repository in a named file" + "fossil init {{path/to/filename}}" taken together helps to sew together the bigger picture idea that a repository is in fact a file(name).  But I'm happy to change it if you do not agree.

**Comment by dz-at-tc** [comment]
So given my comments above, and given @kbdharun 's thumbs up on them, I'm going to assume that this PR is done / ready to merge unless someone disagrees and says otherwise, thanks!

File: pages/common/fossil-init.md (hunk 1 above)
Comment (mfrw) [reply pages/common/fossil-init.md:9]: Today i learnt, fossil does things differently

**Review by mfrw — APPROVED** [review]
Thank you :)
And yes, I nitpick a lot :(
I hope you don't mind :)

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**